Homework2/src/Main.java

```diff
@@ -0,0 +1,164 @@
+import java.util.Arrays;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        int[] test = new int[]{1, 1, 0, 0, 1, 0, 1, 1, 0, 0};
+        int[] empty = new int[8];
+        int[] bnum = new int[]{1, 5, 3, 2, 11, 4, 5, 2, 4, 8, 9, 1};
+        int[] arr = new int[]{1, 5, 3, 2, 11, 4, 5, 2, 4, 8, 9, 1};
+        int[] ohSoUniqueArr = new int[]{10, 10, 5, 5, 5, 5, 10};
+        int[][] matrixHasYou = negArray();
+        int[] moveArr = new int[]{1, 2, 3, 4};
+
+
+//        System.out.println(Arrays.toString(invert(test)));
+//        System.out.println(Arrays.toString(fillUp(empty)));
+//        System.out.println(Arrays.toString(multiply(bnum)));
+//        minMax(arr);
+//        System.out.println(sumTrue(ohSoUniqueArr));
+//        printArray(matrixHasYou);
+//        moveArray(moveArr, 3);
+    }
+
+
+    /**
+     * Первое задание
+     */
+
+    public static int[] invert(int[] a) {
+        for (int i = 0; i < a.length; i++) {
+            if (a[i] == 1) {
+                a[i] = 0;
+            } else {
+                a[i] = 1;
+            }
+        }
+        return a;
+    }
+
+    /**
+     * Второе задание
+     */
+
+    public static int[] fillUp(int[] a){
+        int i = 0;
+        for (int j = 0; j <= 21; j+=3){
+            a[i] = j;
+            i++;
+        }
+        return a;
+    }
+
+    /**
+     * Третье задание
+     */
+
+    public static int[] multiply(int[] a) {
+        for (int i = 0; i < a.length; i++) {
+            if (a[i] < 6){
+                a[i] *= 2;
+            }
+        }
+        return a;
+    }
+
+    /**
+     * Четвертое задание
+     */
+
+    static int[][] negArray() {
+        int[][] a = new int[4][4];
+        for (int i = a.length -1 ; i >= 0; i--) {
+            for (int j = a[i].length - 1; j >= 0; j--) {
+                a[i][j] = i - j + 1;
```

Comment:
```suggestion
                a[i][j] = 1;
```
явное лучше неявного